code/Modules/Input/Core/InputEnums.h

```diff
@@ -32,29 +32,29 @@ struct PointerLockMode {
 /// gamepad buttons, triggers, sticks
 struct GamepadGizmo {
     enum Code {
-        A = 0,
-        B,
-        X,
-        Y,
-        
-        Start,
-        Back,
-        
-        RightBumper,
-        LeftBumper,
-        
-        DPadLeft,
-        DPadRight,
-        DPadUp,
-        DPadDown,
-        
-        LeftTrigger,
-        RightTrigger,
-        LeftStick,
-        RightStick,
-        
-        NumGamepadGizmos,
-        InvalidGamepadGizmo,
+        A                   = (1 << 0),
+        B                   = (1 << 1),
+        X                   = (1 << 2),
+        Y                   = (1 << 3),
+
+        Start               = (1 << 4),
+        Back                = (1 << 5),
+
+        RightBumper         = (1 << 6),
+        LeftBumper          = (1 << 7),
+
+        DPadLeft            = (1 << 8),
+        DPadRight           = (1 << 9),
+        DPadUp              = (1 << 10),
+        DPadDown            = (1 << 11),
+
+        LeftTrigger         = (1 << 12),
+        RightTrigger        = (1 << 13),
+        LeftStick           = (1 << 14),
+        RightStick          = (1 << 15),
+
+        NumGamepadGizmos    = (1 << 16),
```

Comment:
Since the gizmos are now flag bits, the NumGamepadGizmos item should be removed (there are also various o_assert_range_dbg checks which should be removed).